internal/assets/skills/chained-pr/references/chaining-details.md

```
@@ -1,99 +1,78 @@
-- [ ] This PR can be rolled back without unrelated changes
-- [ ] Tests, docs, or manual verification cover this unit
-```
-
-## Commands
-
-```bash
-gh pr view <PR_NUMBER> --json additions,deletions,changedFiles,title,url
-gh pr create --base feat/my-feature --title "feat(scope): focused slice" --body-file pr-body.md
-gh pr create --base feat/my-feature-01-core --title "feat(scope): next focused slice" --body-file pr-body.md
+- Rollback: <independent rollback boundary>
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)